Dockerfile

```diff
@@ -0,0 +1,167 @@
+FROM debian:buster
```

Comment:
You could probably use

`FROM hathitrust/feed-base:buster` and pull in most of this stuff (along with some dependencies you don't need)

Eventually we should make a common base image for HT web apps

Reply:
I started with hathitrust/feed-base:buster and then walked it back to something I had more fine-grained control over. Mostly to get a feel for what was happening behind the scenes. I'll stick with that for now as I get a handle on whether and what we have in terms of weirder dependencies.

Note also that the effort I have thus far doesn't include any webserver. I flailed around with that for a while and then gave up as I have zero experience with Apache/nginx/whatever config.

I may write a ticket for Travis-izing the tests. What is there is just barely enough for a go/no-go, and it probably would have saved my butt a couple times in the last decade if I had had such.

Reply:
Sounds good. I'm interested in getting something that can replicate development end-to-end on a local workstation, including something more robust to impersonate Shibboleth identities in development. I'm not sure whether it makes sense to fold that in with some of the download module work or put it forth as a separate project in half-cycle planning, but either way it's something I'd like to move forward with.